Sofa/Component/Visual/src/sofa/component/visual/LineAxis.cpp

```diff
@@ -29,11 +29,8 @@
 namespace sofa::component::visual
 {
 
-constexpr const char * lineAxisDeprecatedName = "OglLineAxis";
-
 int LineAxisClass = core::RegisterObject("Display scene axis")
         .add< LineAxis >()
-        .addAlias(lineAxisDeprecatedName)
```

Comment:
It makes me thinking that we dont have a mechanism to warn the user at run-time that one alias has been deleted (and to tell him to use  `LineAxis` in this example)
Maybe adding a "Renamed" in `sofa::helper::lifecycle::ComponentChange` ?